### DISCUSSION TITLE
How to set the position of a cover from within Home Assistant

### DISCUSSION THREAD

**Discussion by pvoncken** [dawidchyrzynski/arduino-home-assistant#121] [Q&A]
Is it possible to set the position of a cover (30% for example) from within HA?
I succeeded in opening, closing and stopping the cover and also to read its position in HA but i cannot find how to set a dedicated postion of ...% in HA.

Help me please.

P.S. great library i'm using it it in 5 of my projects.

**Comment by QuickSander** [comment]
I'm in need of the same feature for my curtains. Therefor I'm planning to add this in a fork and will attempt to add it in this repo via a PR.

**Comment by albpara** [comment]
So sad this feature is not yet supportes or implemented, I am also interested on this!
